presentation-api/src/main/java/org/cytoscape/view/presentation/property/table/CellBarChart.java

```diff
@@ -0,0 +1,7 @@
+package org.cytoscape.view.presentation.property.table;
+
+public interface CellBarChart {
+	
+	// ???
```

Comment:
The implementation of a chart doesn't really matter. Note that I'm not using custom graphics for this, I'm not sure its necessary.

Reply:
Perhaps, but other things might require custom graphics (for example, drawing 2D SMILES structures).